### ISSUE TITLE
After Shut down ZeroNet, obligation to force quit

### ISSUE THREAD

**Issue by numidark** [HelloZeroNet/ZeroNet-mac#1]
After Shut down ZeroNet, obligation to «force quit» ZeroNet deamon.

![issues1](https://cloud.githubusercontent.com/assets/14799472/22073386/7e98251c-dda5-11e6-993a-c75713b5cc6c.png)

**Comment by HelloZeroNet** [comment]
Yeah I also noticed, I will try to find a solution, but until then you can use right click on dock icon > quit
